package.json

```diff
@@ -6,6 +6,7 @@
   "types": "src/js",
   "scripts": {
     "start": "node node_modules/react-native/local-cli/cli.js start",
+    "prepare": "npm run build",
```

Comment:
what's the purpose of this script?

Reply:
Adding it allowed me to install from this branch directly without a published release.